### PR TITLE
Philip/wal 61 buy ui clean up

### PR DESCRIPTION
**Pull request by plondon**
## Description (optional)
A big one, to allow for fiat or crypto input amounts.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Comment by jovi5513** [comment]
1

**Comment by milan-bc** [comment]
In figma files we have different icon then here on page, is this intentional?

image

File: packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts [hunk 1]
```diff
@@ -204,24 +204,44 @@ export default ({
...
+
+      // TODO: rewrite this 🤯
+      const output =
+        orderType === 'SELL'
+          ? fix === 'FIAT'
+            ? { amount, symbol: outputCurrency }
+            : { symbol: outputCurrency }
+          : fix === 'FIAT'
+          ? { symbol: outputCurrency }
+          : { amount, symbol: outputCurrency }
```
Comment (milan-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts:235]: 🤯

**Review by milan-bc — APPROVED** [review]
It looks ok and works as it should, I mentioned found switch icon is diff then Figma, but rest is ok.

**Comment by jovi5513** [comment]
4

**Pull request merged**